nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -233,6 +251,34 @@ public NullnessHint onDataflowVisitMethodInvocation(
     }
   }
 
+  /**
+   * Check if the given symbol is a field that is marked as nullable in any of our library models.
+   *
+   * @param symbol The symbol to check.
+   * @return True if the symbol is a field that is marked as nullable in any of our library models.
+   */
+  private boolean isNullableFieldInLibraryModels(@Nullable Symbol symbol) {
+    if (symbol instanceof Symbol.VarSymbol && symbol.getKind().isField()) {
```

Comment:
Let's add a bailout check here: if `libraryModels.nullableFields().isEmpty()`, immediately return `false`.  That should mitigate any performance impact for current use cases.

Reply:
Added in 819f75e48fa87ef50eae02ec008fc19fbdfbcadf